.github/workflows/pyaleph-ci.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: PyAleph
+
+on:
+  push:
+    branches:
+      - dev
+      - master
+  pull_request:
+    branches:
+      - dev
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # Fetch the whole history for all tags and branches (required for aleph.__version__)
+        fetch-depth: 0
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
```

Comment:
Is this better than using the distribution's packages via `apt` ?

Reply:
Yes, because you can specify the exact version of Python.

Reply:
And in the future test with multiple versions of Python.